app/case/forms.py

```diff
@@ -27,8 +27,6 @@ class Meta:
 
     def clean_email(self):
         email = self.cleaned_data["email"]
-        if User.objects.filter(email=email).exists():
```

Comment:
This is the line that prevents us inviting existing Users successfully, and giving them MS Account, have taken it away and checking for existing Users upstream in the account_details_view (controller)